src/common/libkvs/kvs_lookup.c

```diff
@@ -37,35 +37,48 @@
 
 struct lookup_ctx {
     int flags;
+    flux_t *h;
+    char *key;
+    char *atref;
+
     json_t *treeobj;
     char *treeobj_str; // json_dumps of tree object returned from lookup
     void *val_data;    // result of base64 decode of val object data
     int val_len;
     bool val_valid;
     json_t *val_obj;
+    flux_kvsdir_t *dir;
 };
 
 static const char *auxkey = "flux::lookup_ctx";
 
 static void free_ctx (struct lookup_ctx *ctx)
 {
     if (ctx) {
+        free (ctx->key);
         free (ctx->treeobj_str);
```

Comment:
should we free ```atref``` too?  Can't see where else it is freed.

Reply:
Yes good catch!  I'll fix that.